include/mata/nfa.hh

```diff
@@ -141,6 +148,12 @@ struct Move {
     Move(Symbol symbolOnTransition, const StateSet& states_to) :
             symbol(symbolOnTransition), targets(states_to) {}
 
+    //Trying to make Move movable, does it make sense?
+    Move(Move&& rhs) = default;
+    Move(const Move& rhs) = default;
+    Move & operator=(Move&& rhs) = default;
+    Move & operator=(const Move& rhs) = default;
+
```

Comment:
It should be OK. I have a manual implementation of some of the move operations which seem to be more performant for some reason (could there be some additional operations in the default implementation?).

We can either merge these move implementations, or we can first merge my implementation of move operations. We should remove the comment above either way.

Reply:
Sure, interesting, lets use yours then.